components/dashboard/src/projects/NewProject.tsx

```diff
@@ -348,49 +333,42 @@ export default function NewProject() {
                     )}
                     {loaded && noReposAvailable && isGitHub() && (<div>
                         <div className="px-12 py-20 text-center text-gray-500 bg-gray-50 dark:bg-gray-800 rounded-xl">
-                            <img src={NoAccess} title="No Access" className="m-auto mb-4" />
-                            <h3 className="text-center text-gray-600 dark:text-gray-50 pb-3 font-bold">
-                                No Access
-                            </h3>
                             <span className="dark:text-gray-400">
-                                Authorize GitHub (github.com) or select a different account.
+                                Additional authorization is required to allow access to your repositories and trigger prebuilds for every new push to any of your branches.
```

Comment:
This text is a little long and on GitHub we will only trigger **all** branches if you configure that in .gitpod.yml. Let's adjust wording as [spec'ed](https://github.com/gitpod-io/gitpod/issues/6875#issue-1062916651)

`Additional authorization is
required for our GitHub App to
watch your repositories and
trigger prebuilds.`